packages/gafl-webapp-service/src/handlers/__tests__/__snapshots__/page-handler.spec.js.snap

```diff
@@ -13,7 +13,7 @@ exports[`The page handler function sets the value of pageData with displayAnalyt
         "backRef": null,
         "displayAnalytics": false,
         "mssgs": undefined,
-        "pageLanguageSetToWelsh": false,
+        "pageLanguageSetToWelsh": undefined,
```

Comment:
I think generally snapshots shouldn't have undefined in them - could we set a default for the mock so that this doesn't happen? Although if you amend so that the function isn't extracted anymore (see my comment below) then this may well change in any case.